zebra-rpc/src/methods/get_block_template_rpcs/constants.rs

```diff
@@ -26,8 +26,7 @@ pub const GET_BLOCK_TEMPLATE_MUTABLE_FIELD: &[&str] = &[
 ];
 
 /// A hardcoded list of Zebra's getblocktemplate RPC capabilities.
-/// Currently empty.
```

Comment:
Nitpick: Would you mind linking to the spec for this field, and the other constants before and after it?

https://en.bitcoin.it/wiki/BIP_0023#Mutations
https://en.bitcoin.it/wiki/BIP_0023#Block_Proposal
https://zips.z.cash/protocol/protocol.pdf#blockheader